presto-hive/src/test/java/io/prestosql/plugin/hive/rubix/TestRubixCaching.java

```diff
@@ -213,6 +215,10 @@ private FileSystem getCachingFileSystem(HdfsContext context, Path path)
                         (dynamicConfig, ignoredContext, ignoredUri) -> {
                             dynamicConfig.set("fs.file.impl", CachingLocalFileSystem.class.getName());
                             dynamicConfig.set("fs.azure.account.key", "Zm9vCg==");
+                            dynamicConfig.setBoolean("fs.gs.lazy.init.enable", true);
```

Comment:
move `fs.gs.lazy.init.enable` below entries related to `adl`. Let's group by corporation ;P